client/js/components/procedure/admin/NewBlueprintForm.vue

```diff
@@ -35,49 +35,47 @@
       <legend
         class="sr-only"
         v-text="Translator.trans('blueprint.data')" />
-      <dp-form-row class="u-mb-0_75">
-        <dp-input
-          id="r_name"
-          data-cy="newMasterName"
-          :label="{
-            text: Translator.trans('name')
-          }"
-          maxlength="200"
-          name="r_name"
-          required />
-      </dp-form-row>
+      <dp-input
+        class="mb-0.5"
```

Comment:
I'd suggest that we settle on a method for vertical spacing: either apply a top- or bottom margin, never both. What do you think?

Reply:
Yeah. I thought something similar several times.
I would say, lets always append top-spacing, with that we don't have to handle the last Element in a loop. 
On the other hand, most defaults add spacing to the bottom :thinking: 

What Do You prefer?

Reply:
as @hwiem also mentioned in the related slack discussion, i'd prefer placing the spacing class to the container, and in cases where this is not possible, add top-spacing to elements (as in `<element class="mt-0.5">`).

Reply:
since the spacing was a bit off anyway, I tried to clean it a bit instead of reworking "everything".
52f41f0bdb705a4fb3f635d9ea368d47c0762e55